services/dy-jupyter-extensions/neuron/Dockerfile

```diff
@@ -0,0 +1,95 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} AS common
+LABEL maintainer="guidon"
+
+USER root
+
+# NEURON requirements
+RUN apt-get update &&\
+  apt-get install -y \
```

Comment:
it seems also **--no-install-recommends** is also a recommendation to not install too many additional stuff

Reply:
I added the flag, however, in this case it has no effect...